multibuildingdetector/readers/XMLReader.py

```diff
@@ -1,8 +1,13 @@
+from os import listdir
 import numpy as np
+from os.path import isfile, join
 import xml.etree.ElementTree as ET
-import chainer
+from sklearn.model_selection import train_test_split
 from chainercv.utils import read_image
 
+from .ImageBoundingBoxDataset import ImageBoundingBoxDataset
+
+
 LABEL_NAMES = ('other',
```

Comment:
Since we have now separated dataset and reader, I think the label_names belong to the dataset, hence we can put this in the dataset file.

Reply:
But the labels are specific to what we have and how we read it, hence the it's in the reader.

Therefore other implementation can use the same dataset, but a different reader.

Reply:
I see but we could just return `train, test` in the reader class and instantiate the `ImageBoundingBoxDataset` class in `__main__`. This way the users can add new reader classes as long as they implement something for the Dataset and you can add it in the yml config.